crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -129,19 +129,19 @@ def check_all_repos
         repochecks
       end
 
-      def provided_and_enabled?(feature, platform = nil, arch = nil)
+      def provided_and_enabled?(feature, platform = nil, arch = nil, check_enabled = true)
```

Comment:
I'm somehow a bit uneasy with the fact that we allow disabling a feature of this method that is in the name of the method. I don't know if that's okay or not from a style perspective. (An alternative would be to move the content of this method to a helper and call the helper from here).